iModelCore/ecobjects/src/SchemaMerger.cpp

```diff
@@ -277,6 +317,13 @@ BentleyStatus SchemaMerger::MergeSchemas(SchemaMergeResult& result, bvector<ECSc
     if(failedToFillSchemas)
         return BentleyStatus::ERROR;
 
+     if(SchemaMerger::ValidateUniqueSchemaNames(result.GetResults()) != BentleyStatus::SUCCESS)
+        {
+        result.Issues().ReportV(IssueSeverity::Fatal, IssueCategory::BusinessProperties, IssueType::ECSchema, ECIssueId::EC_0060,

```

Comment:
The ECIssueIds should be unique for every log message call. So you will have to add more and not just reuse 0060 for multiple similar issues

Reply:
The reworked behavior will only require one issue.